src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1279,7 +1306,7 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
     <value>Let's get started!</value>
   </data>
   <data name="Oobe_PowerToysDescription.Text" xml:space="preserve">
-    <value>Welcome to PowerToys!  These overviews will help you quickly learn the basics of all our utilities.</value>
+    <value>Welcome to PowerToys! These overviews will help you quickly learn the basics of all our utilities.</value>
```

Comment:
"These overviews" .. maybe we "This overview" sounds better?

Reply:
Oobe is off-topic